core-api/src/main/java/com/optimizely/ab/Optimizely.java

```diff
@@ -127,7 +128,12 @@ private Optimizely(@Nonnull EventHandler eventHandler,
             if (getProjectConfig() != null) {
```

Comment:
It looks like my previous comment on this line got lost :)
We discussed this blocking "getProjectConfig" while constructing Optimizely is not desirable, so we need to make it return null if not available. @msohailhussain may have a good idea on how to fix it.